src/containers/TasksByDayPage/View.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+
+import Box from '@material-ui/core/Box'
+import Container from '@material-ui/core/Container'
+import Tab from '@material-ui/core/Tab'
+import Tabs from '@material-ui/core/Tabs'
+import styled from 'styled-components'
+
+import ActionButton from './components/ActionButton'
+import TabContainer from './components/TabContainer'
+import TabContent from './components/TabContent'
+
+export default function View({ activeIndex, onChange }) {
+  return (
+    <StyledContainer maxWidth="md">
+      <div className="marginTop">
+        <ActionButton />
+        <Box display="flex" className="marginTop">
```

Comment:
```suggestion
        <Box display="flex" className="margin-top">
```
tên class cho css dùng dash case a nhé :v (note: mà thực ra cái này ko nên reuse, vì nó ko liên quan, tần suất code css cũng sẽ sửa thường xuyên hơn code server, nên duplicate tốt hơn là abstract sai :v)